quandl/util.py

```diff
@@ -60,7 +60,16 @@ def convert_to_date(value):
             return value
 
     @staticmethod
-    def convert_options(**options):
+    def convert_options(request_type, **options):
+        if request_type == 'get':
+            return Util._convert_options_for_get_request(**options)
+        elif request_type == 'post':
+            return Util._convert_options_for_post_request(**options)
```

Comment:
would it be easy to create a `_convert_options_for_request` and pass in different parameter(get, post) to use different handling logic?

Reply:
Your suggestion is technically what is implemented here except the `_convert_options_for_request` function would have the `if GET elif POST` logic. No matter what we'll need to have separate logic for the two request methods and we'll need to have an if statement somewhere to determine which to call - don't think it matters where this code resides so I think it makes sense to keep this function the way it is.